<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set_global_timezone</class>
 <widget class="QWidget" name="set_global_timezone">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>157</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>386</width>
    <height>157</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>386</width>
    <height>157</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Timezone Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_start_date">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>92</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>92</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>From</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateEdit_start">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_start_date_2">
       <property name="minimumSize">
        <size>
         <width>92</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>92</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateEdit_end">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_time_zone">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>96</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Time Zone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_timezone">
       <property name="minimumSize">
        <size>
         <width>270</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>5445454</width>
         <height>26</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>78</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
